### DISCUSSION TITLE
A html Tag is missing on your new WebSite

### DISCUSSION THREAD

**Discussion by Silbad** [directus/directus#3214] [Archive]
Hello,

Very nice your new website, just a little thing ;)

![image](https://user-images.githubusercontent.com/31576982/100426878-db25a280-3091-11eb-8a10-4896d8e8d289.png)

![image](https://user-images.githubusercontent.com/31576982/100427347-9b12ef80-3092-11eb-9695-b323122c6824.png)

**Comment by benhaynes** [comment]
Good eye! Thanks for the heads up... we've fixed this. 👍
